packages/sdk/src/utils/LocalConfigurationDecorator.ts

```diff
@@ -0,0 +1,68 @@
+import { StudioOptionsDeltaUpdate, WellKnownConfigurationKeys } from '../types/ConfigurationTypes';
+import { getEditorResponseData } from './EditorResponseData';
+import { ConfigurationController } from '../controllers/ConfigurationController';
+
+export class LocalConfigurationDecorator {
+    /**
+     * @ignore
+     */
+    #localConfig: Map<string, string>;
+
+    /**
+     * @ignore
+     */
+    #configurationController: ConfigurationController;
+
+    /**
+     * @ignore
+     */
+    constructor(configurationController: ConfigurationController, localConfig: Map<string, string>) {
+        this.#configurationController = configurationController;
+        this.#localConfig = localConfig;
+    }
+
+    /**
+     * This method returns the value for a given configuration key. If a value was not found in the configuration store
+     * this method returns an error. The key cannot be null.
+     * @returns value for a given configuration key
+     */
+    getValue = async (key: WellKnownConfigurationKeys | string) => {
+        const value = this.#localConfig.get(key);
+        if (value) {
```

Comment:
An undefined check would be better

Reply:
it is undefined check or WDYM?

Reply:
`value` can contain an empty string which is also valid

Reply:
```suggestion
        if (value != null) {
```